src/scss/design-system-interface.scss

```diff
@@ -156,13 +156,17 @@ $c-ame-orange: #F26531;
 
   &__sub {
     font-family: $primary-font;
-    font-size: 16px;
+    font-size: 1rem;
```

Comment:
We could also use a `font-size` variable system as well as mentioned [ here](https://github.com/meranicosme/a-modern-eden/pull/1/files#r323415179)